tools/x-workbench/package.json

```diff
@@ -5,7 +5,7 @@
   "description": "",
   "main": "register-components.js",
   "scripts": {
-    "start": "start-storybook -p 9001 -c .storybook -s static -h local.ft.com",
```

Comment:
Please leave this in, there are some components that will fail due to CORS issues without it.